app/code/Magento/Checkout/view/frontend/web/js/checkout-data.js

```diff
@@ -33,20 +33,24 @@ define([
 
     if ($.isEmptyObject(getData())) {
         checkoutData = {
-            'selectedShippingAddress': null,
-            'shippingAddressFromData': null,
-            'newCustomerShippingAddress': null,
-            'selectedShippingRate': null,
-            'selectedPaymentMethod': null,
-            'selectedBillingAddress': null,
-            'billingAddressFormData': null,
-            'newCustomerBillingAddress': null
+            'selectedShippingAddress': null, // Selected shipping address pullled from local storage (Persistence)
```

Comment:
1. remove space between // and Selected (fix all cases)
2. replace local storage (Persistence) and  local storage --> persistence storage